moxie/cores/ssh.py

```diff
@@ -76,7 +76,7 @@ def exit(stdin, stdout, stderr, args=None):
 def readl(stdin, stdout, echo=True):
     buf = ""
     while not stdin.at_eof():
-        bytes_ = (yield from stdin.read())
+        bytes_ = (yield from stdin.read(1))
```

Comment:
The interface would hang as soon as I connected, so this was the fix I put in.  I'll take it down and mess with it a little before resubmitting.